packages/web-app-files/src/components/SideBar/Links/FileLinks.vue

```diff
@@ -55,11 +53,7 @@
       </template>
     </div>
     <div v-if="currentView === VIEW_EDIT" :key="VIEW_EDIT">
-      <transition
-        enter-active-class="uk-animation-slide-right uk-animation-fast"
-        leave-active-class="uk-animation-slide-right uk-animation-reverse uk-animation-fast"
-        name="custom-classes-transition"
-      >
+      <transition name="custom-classes-transition">
```

Comment:
The list transitions are from older times. You can drop the transition tag entirely for now. We'll have to come up with something nicer.